analyzer/src/main/kotlin/managers/PIP.kt

```diff
@@ -74,6 +74,19 @@ object VirtualEnv : CommandLineTool {
     override fun getVersionRequirement(): Requirement = Requirement.buildIvy("15.1.+")
 }
 
+object ProjectVersion : CommandLineTool {
+    override fun command(workingDir: File?) = if (OS.isWindows) "python" else "python3"
+
+    fun isPython3(workingDir: File): Boolean{
```

Comment:
Could you please add some function docs saying that this will return `true` only if *all* `.py` files found in the `workingDir` can be compiled with Python 3? Also, we should make clear that this doe snot not mean the project *requires* Python 3, but that it is fully compatible with Python 3 (and maybe also with Python 2). Correct?

Reply:
- [x] Add docs.